src/view/treeNodes/pullRequestNode.ts

```diff
@@ -564,7 +564,12 @@ export class PRNode extends TreeNode {
 
 	private async editComment(document: vscode.TextDocument, comment: vscode.Comment, text: string): Promise<void> {
 		const fileChange = this.findMatchingFileNode(document.uri);
-		const rawComment = await this._prManager.editReviewComment(this.pullRequestModel, comment.commentId, text);
+		const existingComment = fileChange.comments.find(c => c.id.toString() === comment.commentId);
```

Comment:
May as well leverage what the `==` operator is best at ;)

```suggestion
		const existingComment = fileChange.comments.find(c => c.id == comment.commentId);
```

Reply:
Good point! It seems we have a `tslint` rule against `==` though, so I think I'll keep the cast explicit